portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,10 +37,10 @@
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
-        // If the quantity is null it will be covered. Null values will happen when the user leave the quantity input blank. 
-
-        Integer quantity = Integer.parseInt(request.getParameter("quantity"));
-
+        // If the quantity is null it will be covered by the default qaunity. Null values will happen when the user leave the quantity input blank. 
+        int defaultQuantity = 5;
```

Comment:
nit: make this a constant, and follow style guide `DEFAULT_QUANTITY` (https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names)